composeApp/src/commonMain/kotlin/sp/bvantur/tasky/event/data/local/EventLocalDataSource.kt

```diff
@@ -47,9 +40,30 @@ class EventLocalDataSource(
                 database.getEventDao().insert(event)
                 database.getAttendeeDao().insertItems(attendees)
             }
+            database.invalidationTracker.refreshAsync()
         }
         TaskyResult.Success(Unit).asEmptyDataResult()
     } catch (ignore: SQLiteException) {
         TaskyResult.Error(TaskyError.SqlError)
     }
+
+    suspend fun getEventWithAttendeesById(
+        eventId: String
+    ): TaskyResult<Pair<EventEntity?, List<AttendeeEntity>>, TaskyError> {
+        try {
+            var event: EventEntity? = null
+            var attendees: List<AttendeeEntity> = emptyList()
+            database.useWriterConnection {
+                it.immediateTransaction {
+                    event = database.getEventDao().getEventById(eventId)
+                    attendees = database.getAttendeeDao().getAttendeesByEventId(eventId)
+                }
+                database.invalidationTracker.refreshAsync()
```

Comment:
Is there a specific reason you want to trigger a refresh here manually? You'll get notified of any changes when you return a Flow from the database automatically.

Reply:
There is a bug in Room for KMP where this call is needed in order for a Flow to pickup DELETE changes in my project. I can add TODO to remove it when it will be fixed in Room library.